test/test_metrics.py

```diff
@@ -196,6 +196,47 @@ def test_metric(self, device, dtype):
         assert_close(actual, expected)
 
 
+class TestAepe:
+    def test_metric_mean_reduction(self, device, dtype):
+        sample = torch.ones(4, 4, 2, device=device, dtype=dtype)
+        expected = torch.tensor(0.565685424, device=device, dtype=dtype)
+        actual = kornia.metrics.aepe(sample, 1.4 * sample, reduction="mean")
+        assert_close(actual, expected)
+
+    def test_metric_sum_reduction(self, device, dtype):
+        sample = torch.ones(4, 4, 2, device=device, dtype=dtype)
+        expected = torch.tensor(1.4142, device=device, dtype=dtype) * 4**2
+        actual = kornia.metrics.aepe(sample, 2.0 * sample, reduction="sum")
+        assert_close(actual, expected)
+
+    def test_metric_no_reduction(self, device, dtype):
+        sample = torch.ones(4, 4, 2, device=device, dtype=dtype)
+        expected = torch.zeros(4, 4, device=device, dtype=dtype) + 1.4142
+        actual = kornia.metrics.aepe(sample, 2.0 * sample, reduction="none")
+        assert_close(actual, expected)
+
+    def test_metric_wrong_reduction(self, device, dtype):
+        with pytest.raises(NotImplementedError):
```

Comment:
you didn't need to provide test for NotImplemented raises, we consider it as a "defensive code"